src/Microsoft.DotNet.VersionTools/Dependencies/IDependencyUpdater.cs

```diff
@@ -14,6 +14,6 @@ public interface IDependencyUpdater
         /// <summary>
         /// Updates based on the given build infos and returns build infos used during update.
         /// </summary>
-        IEnumerable<BuildInfo> Update(IEnumerable<BuildInfo> buildInfos);
+        IEnumerable<DependencyUpdateTask> Update(IEnumerable<BuildInfo> buildInfos);
```

Comment:
Is it weird to call this "Update" when this method actually doesn't do the "Update"?  You need to start the tasks in order to do the update.